### PR TITLE
Inequitable share frontend

### PR DESCRIPTION
**Pull request by joshzarrabi**
## Preview Link

<a href="https://deploy-preview-1838--health-equity-tracker.netlify.app/exploredata?onboard=false&mls=1.covid-3.02&demo=sex&dt1=covid_deaths#inequities-over-time" target="_blank">
  <br />
Click to demo updates </a> 

## Description

frontend for #1829 

## To Do
- [x] fix high/low default on ShareTrends
- [x] the metric isn't really a share... it's like a "pct_deviation" not sure if it's worth renaming the metric on the backend now? I just don't want to confuse ourselves thinking it represents a share of total cases, it's really "by what percent is the result off from the expected"
- [x] fine-tooth comb to confirm all labels, etc are transferred. We really still need to do #1438 because it's growing more and more confusing and the new title generating functions are kind of crazy. There should be more 1:1 of strings of text in the metric config and the text that appears on the page, and less string manipulation to slice and dice and end up with the right phrasing

## Screenshots (if appropriate):

Screen Shot 2022-10-11 at 9 37 01 PM

## Types of changes

- New feature (non-breaking change which adds functionality or content)

### CODE REVIEW

**Comment by benhammondmusic** [comment]
Screen Shot 2022-10-11 at 9 50 05 PM

File: frontend/src/data/config/MetricConfig.ts [hunk 1]
```diff
@@ -333,11 +359,8 @@ export const METRIC_CONFIG: Record<DropdownVarId, VariableConfig[]> = {
         pct_share: {
           metricId: "covid_cases_share",
           fullCardTitleName: "Share of total COVID-19 cases",
-          chartTitle: "Share of total COVID-19 cases with unknown",
```
Comment (eriwarr) [inline frontend/src/data/config/MetricConfig.ts:336]: Removing the _chartTitle_  prop will cause the chart title on the Unknown maps to be partially blank.

**Review by eriwarr — CHANGES_REQUESTED** [review]
This is the only thing I noticed. Everything else looks great!!

Unknown Map

**Comment by joshzarrabi** [comment]
Looks good to me! Just update the col name to https://github.com/SatcherInstitute/health-equity-tracker/pull/1857 and we should be good to go.

**Review by eriwarr — APPROVED** [review]
Great job Ben!!!

**Pull request merged**